src/main/java/seedu/recruittrackpro/ui/MainWindow.java

```diff
@@ -71,12 +79,42 @@ public MainWindow(Stage primaryStage, Logic logic) {
         setAccelerators();
 
         helpWindow = new HelpWindow();
+
+        toggleTheme(); // Sets initial theme to dark
     }
 
     public Stage getPrimaryStage() {
         return primaryStage;
     }
 
+    @FXML
+    private void initialize() {
+        themeToggle.setOnAction(e -> toggleTheme());
+    }
+
+    private void toggleTheme() {
+        Scene scene = getRoot().getScene();
+        ObservableList<String> stylesheets = scene.getStylesheets();
+        String newTheme;
+
+        stylesheets.removeIf(sheet ->
+                sheet.contains("DarkTheme.css") || sheet.contains("LightTheme.css")
+        );
+
+        if (isLightMode) {
+            newTheme = getClass().getResource(LIGHT_THEME).toExternalForm();
+            themeToggle.setText("☾");
+        } else {
+            newTheme = getClass().getResource(DARK_THEME).toExternalForm();
+            themeToggle.setText("☼");
+        }
```

Comment:
The icons do not render as nice on Windows, compared to the screenshot on Mac.

I tried using [this](https://stackoverflow.com/a/62922094) and it looks more similar to the one on Mac.

```suggestion
        if (isLightMode) {
            newTheme = getClass().getResource(LIGHT_THEME).toExternalForm();
            themeToggle.setText(new String(Character.toChars(0x1F319)));
        } else {
            newTheme = getClass().getResource(DARK_THEME).toExternalForm();
            themeToggle.setText(new String(Character.toChars(0x2600)));
        }
```